cmd/kube-proxy/app/server.go

```diff
@@ -104,11 +112,15 @@ type Options struct {
 	master string
 	// healthzPort is the port to be used by the healthz server.
 	healthzPort int32
+
+	scheme *runtime.Scheme
```

Comment:
Having this in the Options is strange. If it is just for writing, can't be create them locally in the write func?